css/grid.styl

```diff
@@ -0,0 +1,227 @@
+// GRID
+
+.container
+  margin: 0 auto
+  max-width: 2400px 
+  
+.grid-row
+  box-sizing: border-box
+  display: flex
+  flex-direction: row
+  flex-wrap: wrap
+  &.reverse
+    flex-direction: row-reverse
+    
+.grid-column
+  box-sizing: border-box
+  display: flex
+  flex-direction: column
+  &.reverse
+    flex-direction: column-reverse
+    
+.grid-item
+  position: relative
+  flex: 0 1 auto
+  box-sizing: border-box
+  max-width: 100%
+  max-height: 100%
+
+.no-gutter
+  padding-left: 0
+  padding-right: 0
+  
+.flex-grow
+  flex-grow: 1
+  
+// FLEX POSITIONING
+
+  //ALIGN CONTENT  
+
+.align-content-start
+  align-content: flex-start
+
+.align-content-center
+  align-content: center
+
+.align-content-end
+  align-content: flex-end
+  
+.align-content-between  
+  align-content: space-between
+
+.align-content-around
+  align-content: space-around
+
+.align-content-stretch
+  align-content: stretch
+
+  //ALIGN ITEMS 
+    
+.align-items-start
+  align-items: flex-start
+
+.align-items-center
+  align-items: center
+
+.align-items-end
+  align-items: flex-end
+  
+.align-items-baseline
+  align-items: baseline
+
+.align-items-stretch
+  align-items: stretch
+  
+  //ALIGN SELF 
+
+.align-self-auto
+  align-self: auto
+  
+.align-self-start
+  align-self: flex-start
+
+.align-self-center
+  align-self: center
+
+.align-self-end
+  align-self: flex-end
+  
+.align-self-baseline
+  align-self: baseline
+
+.align-self-stretch
+  align-self: stretch
+  
+  //JUSTIFY CONTENT 
+
+.justify-start
+  justify-content: flex-start
+
+.justify-center
+  justify-content: center
+
+.justify-end
+  justify-content: flex-end
+  
+.justify-between
+  justify-content: space-between
+  
+.justify-around
+  justify-content: space-around
+  
+$golden-ratio = 1.61803398875
```

Comment:
we dont need the hard value cus we got kouto swiss. We can use the golden ratio scaler or something smaller from here: http://kouto-swiss.io/docs.html#modular-scale. `perfect-fourth` sounds like it might be nice